Scripts/deafrica_datahandling.py

```diff
@@ -478,6 +486,17 @@ def load_ard(dc,
     if requested_measurements:
         ds = ds[requested_measurements]
     
+    # Scale data 0-1 if requested
+    if scaling=='normalised':
+        if product_type == 'c1':
+            ds = ds / 10000
+        if product_type == 's2':
+            ds = ds / 10000   
+    
+    if product_type == 'c2':
+        ds = ds * 2.75e-5 - 0.2
+        print("Scaling Landsat C2 using ds = ds * 2.75e-5 - 0.2")
+    
```

Comment:
Same story as above, need to detect and not scale masking bands.

Reply:
Something like this?
```
if product_type == 'c2':
        print("Scaling Landsat C2 using ds = ds * 2.75e-5 - 0.2")
        
        if 'quality_l2_aerosol' in ds.data_vars:
            ds_data = ds[data_bands]
            ds_masks = ds[mask_bands]
            ds_data = ds_data * 2.75e-5 - 0.2
            ds = xr.merge([ds_data, ds_masks])
            ds.attrs.update(attrs)
        
        else:
            ds = ds * 2.75e-5 - 0.2
```